fastlane/spec/actions_specs/hg_commit_version_bump_spec.rb

```diff
@@ -15,16 +15,6 @@
         expect(result).to eq("hg commit -m 'Version Bump'")
       end
 
-      it 'return the project path when ignoring the cocoapods path' do
-        all_xcodeproj_paths = ["/Users/tmp/Developer/iOS/MyPods/MyProject/MyProject.xcodeproj", "/Users/tmp/Developer/iOS/MyPods/MyProject/Pods/Pods.xcodeproj"]
-        project_path = Fastlane::Actions.ignore_cocoapods_path(all_xcodeproj_paths)
-        expect(project_path).to eq(all_xcodeproj_paths[0..0])
-
-        all_xcodeproj_paths = ["/Users/tmp/Developer/iOS/MyPods/MyProject/MyProject.xcodeproj", "/Pods/SVPullToRefresh/Demo/SVPullToRefreshDemo.xcodeproj"]
-        project_path = Fastlane::Actions.ignore_cocoapods_path(all_xcodeproj_paths)
-        expect(project_path).to eq(all_xcodeproj_paths[0..0])
-      end
-
```

Comment:
Removed this action-specific test for ignoring dependency packages, as this is now `XcodeprojHelper`'s responsibility and tested there.